test/suites/common-tanssi/fees/test_fee_balance_transfer.ts

```diff
@@ -194,7 +194,7 @@ describeSuite({
 
                 const totalSupplyAfter = (await polkadotJs.query.balances.totalIssuance()).toBigInt();
 
-                expect(totalSupplyAfter - totalSupplyBefore).to.equal(issuance - fee);
+                expect(totalSupplyAfter - totalSupplyBefore).to.equal(issuance - (fee * 4n) / 5n);
```

Comment:
We can probably check that the treasury account receives the remaining 20%. The account that receives the tokens is composed by the bytes ` "tns/tsry"` plus appending zeroes.

You can probably create a function that returns this account in util.ts

Reply:
Ah never mind you have a test doing so. Great!